contracts/babylon/src/msg/contract.rs

```diff
@@ -18,6 +18,31 @@ pub trait ContractMsg {
     fn babylon_tag_to_bytes(&self) -> StdResult<Vec<u8>>;
 }
 
+#[cw_serde]
+pub struct BtcLightClientInitMsg {
+    pub btc_light_client_code_id: u64,
+    /// Instantiation message for the BTC light client contract.
+    pub btc_light_client_msg: Option<Binary>,
+}
+
+#[cw_serde]
+pub struct BtcStakingInitMsg {
+    pub btc_staking_code_id: u64,
+    /// Name of the consumer
+    pub consumer_name: String,
+    /// Description of the consumer
+    pub consumer_description: String,
```

Comment:
The concepts `consumer_name` and `consumer_description` exist at system level, not just in the BTC staking contract. So it belongs to Babylon contract (which is the "router" contract)

Reply:
That's fair, then this PR does not make sense.